src/api/types/index.tsx

```diff
@@ -86,6 +86,12 @@ export type FamilyStudentRequest = FamilyRequest & {
   parent: StudentRequest;
 };
 
+export type FamilySessionRequest = {
+  family: FamilyStudentRequest;
+  session: SessionListResponse["id"] | null;
```

Comment:
we shouldn't ever not pass in a session!
```suggestion
  session: number;
```